frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -8,6 +8,15 @@ interface HighlightEllipsisProps {
   unsafeHTML: string;
 }
 
+function applyHTMLFilter(str: string) {
```

Comment:
Oh, also maybe rename `string` to `unsafeHTML` (preferred, IMO) or `htmlString`.